src/test/java/com/vaadin/flow/ai/formfiller/views/FormFillerTextDemo.java

```diff
@@ -331,11 +331,12 @@ public HashMap<String, String> getExampleTexts() {
                 "Ruukinkatu 2-4, FI-20540 Turku (Finland), he is 45 years old and can be reached at phone number 555-1234 " +
                 "and at email 'andrewjackson@gmail.com. Andrew has placed five items: number 1001 " +
                 "contains two items of smartphone for a total of $1,000 placed on 2023 January " +
-                "the 10th with a status of deliberate; number 1002 includes one item of laptop " +
-                "with a total of $1,500 placed on 2023 February the 15th with a status of in transit; " +
+                "the 10th with a status of delivered and delivered the 2023 January the 20th; number 1002 includes one item of laptop " +
```

Comment:
not sure if this is a bit out of English by purpose, this would be more native I guess (I am non native too :D):
```suggestion
                "the 10th with a status of delivered and it was delivered on the 20th of January, 2023; number 1002 includes one item of laptop " +
```

Reply:
Or we can let it be like this as a test if the model can recognize and fetch information properly 👍